ci-operator/config/openstack-k8s-operators/designate-operator/openstack-k8s-operators-designate-operator-main.yaml

```diff
@@ -6,6 +15,22 @@ images:
     RUN pip3 install pre-commit
   from: src
   to: openstack-operator-ci-pre-commit
+- dockerfile_literal: |
+    FROM src
+    RUN pip3 install python-openstackclient
+    COPY oc /usr/bin/oc
+    COPY opm /usr/bin/opm
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    opm-builder:
+      paths:
+      - destination_dir: .
+        source_path: /bin/opm
```

Comment:
sanity check... is that right? you have /usr/bin/opm on line 22

Reply:
thanks and sorry for the extra step. at least now we know it should be this way ;)